scripts/compose.py

```diff
@@ -611,6 +620,12 @@ def add_arguments(cls, parser):
             default=[],
             help="arbitrary additional configuration to set for apm-server"
         )
+        parser.add_argument(
+            "--apm-server-disable-acm",
```

Comment:
can you flip this to `apm-server-acm-disable` to match `--apm-server-ilm-disable` ?